src/speech/speech_synthesis_word_boundary_event.rs

```diff
@@ -35,6 +41,7 @@ impl SpeechSynthesisWordBoundaryEvent {
                     synthesizer_event_handle_release,
                 ),
                 audio_offset,
+                duration_ms,
```

Comment:
api **synthesizer_word_boundary_event_get_values** now returns also **boundary_type**.  Why this new parameter was not added to struct **SpeechSynthesisWordBoundaryEvent** same as **duration_ms**?

Reply:
Good catch looks like I completely forgot about it